component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1801,4 +1834,26 @@ private void publishPostSMSOTPValidatedEvent(HttpServletRequest request,
             throw new AuthenticationFailedException(errorMsg, e);
         }
     }
+
+    /*
+     * This method returns the boolean value of the mobile number update failed context property
+     *
+     * @param context
+     * @return The status of mobile number update failed parameter
+     */
+    private boolean isMobileNumberUpdateFailed (AuthenticationContext context) {
```

Comment:
formatting issue. Removed space after isMobileNumberUpdateFailed

Reply:
Changed the format